.github/workflows/java_ci_with_maven_temurin.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven[temurin]
```

Comment:
really?

I hope it could be clear
<img width="314" alt="image" src="https://user-images.githubusercontent.com/4465723/154616533-a5e61632-a205-4e8f-82b4-204b54b504c1.png">

we can rename it to CI, If we use only adopt